libraries/SITL/SIM_StratoBlimp.cpp

```diff
@@ -0,0 +1,300 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  StratoBlimp simulator class
+*/
+
+#include "SIM_StratoBlimp.h"
+
+#if AP_SIM_STRATOBLIMP_ENABLED
+
+#include <AP_Logger/AP_Logger.h>
+#include <AP_Motors/AP_Motors.h>
+
+#include <stdio.h>
+
+using namespace SITL;
+
+extern const AP_HAL::HAL& hal;
+
+// SITL Ship parameters
+const AP_Param::GroupInfo StratoBlimp::var_info[] = {
+    // @Param: MASS
+    // @DisplayName: mass
+    // @Description: mass of blimp not including lifting gas
+    // @Units: kg
+    AP_GROUPINFO("MASS",     1, StratoBlimp,  mass, 80),
+
+    // @Param: HMASS
+    // @DisplayName: helium mass
+    // @Description: mass of lifting gas
+    // @Units: kg
+    AP_GROUPINFO("HMASS",    2, StratoBlimp,  helium_mass, 13.54),
+
+    // @Param: ARM_LEN
+    // @DisplayName: arm length
+    // @Description: distance from center of mass to one motor
+    // @Units: m
+    AP_GROUPINFO("ARM_LEN",  3, StratoBlimp,  arm_length, 3.6),
+
+    // @Param: MOT_THST
+    // @DisplayName: motor thrust
+    // @Description: thrust at max throttle for one motor
+    // @Units: N
+    AP_GROUPINFO("MOT_THST", 4, StratoBlimp,  motor_thrust, 145),
+
+    // @Param: DRAG_FWD
+    // @DisplayName: drag in forward direction
+    // @Description: drag on X axis
+    AP_GROUPINFO("DRAG_FWD", 5, StratoBlimp,  drag_fwd, 0.27),
+
+    // @Param: DRAG_FWD
```

Comment:
this is a duplicate param name! causing wiki build errors in parameters docs...please fix! should be DRAG_UP I think

Reply:
fixed, thanks